third-parties/ReadMe.txt

```diff
@@ -0,0 +1,35 @@
+SLVS - third-party assemblies folder
```

Comment:
Having explained how this all works in review comments, I realised it would make sense to make a more permanent note of it somewhere nearer the code, so I added this read-me file.